script/nsis/installer.nsi

```diff
@@ -0,0 +1,53 @@
+Name    "Git LFS"
+OutFile "GitLFSInstaller.exe"
+
+
+InstallDir $DESKTOP
+
+VIProductVersion "0.6.0.0"
+VIAddVersionKey ProductName      "Git LFS"
+VIAddVersionKey Comments         "Git LFS tool"
+VIAddVersionKey CompanyName      "GitHub, Inc"
+VIAddVersionKey LegalCopyright   "GitHub, Inc"
+VIAddVersionKey FileDescription  "Git LFS Tool"
+VIAddVersionKey FileVersion      1
+VIAddVersionKey ProductVersion   1
+VIAddVersionKey InternalName     "Git LFS"
+VIAddVersionKey LegalTrademarks  "something, something, darkside"
+VIAddVersionKey OriginalFilename "GitLFSInstaller.exe"
```

Comment:
Yes, I highly recommend EV code signing, Windows 8, 8.1 & 10 all have SmartScreen enabled by default and non-codesigned downloads are usually flagged. http://blogs.msdn.com/b/ie/archive/2012/08/14/microsoft-smartscreen-amp-extended-validation-ev-code-signing-certificates.aspx

In my experience even code signed apps can be flagged as untrusted to begin with until enough people have downloaded them if the cert is new, so if you have a cert you already use for say GitHub Desktop, definitely try to re-use it.